fay.cabal

```diff
@@ -1,5 +1,5 @@
 name:                fay
-version:             0.23.1.16
+version:             0.24
```

Comment:
Is there anything breaking in this release? From the looks of it this could be `0.23.1.17` instead since the API is unaffected.